libraries/bot-builder/src/main/java/com/microsoft/bot/builder/skills/SkillHandlerImpl.java

```diff
@@ -101,6 +118,9 @@ public SkillHandlerImpl(
             callback);
     }
 
+    /**
+     * {@inheritDoc}
+     */
```

Comment:
This class is not extending from another.. so the `inheritDoc` it's not okay. We should populate the commentaries.